src/js/cards/image.js

```diff
@@ -1,96 +1,17 @@
-import placeholderImage from './placeholder-image';
-import { FileUploader } from '../utils/http-utils';
-
-function buildFileInput() {
-  let input = document.createElement('input');
-  input.type = 'file';
-  input.accept = 'image/*';
-  input.classList.add('ck-file-input');
-  document.body.appendChild(input);
-  return input;
-}
-
-function buildButton(text) {
-  let button = document.createElement('button');
-  button.innerHTML = text;
-  return button;
-}
-
-function upload(imageOptions, fileInput, success, failure) {
-  let uploader = new FileUploader({
-    url: imageOptions.uploadUrl,
-    maxFileSize: 5000000
-  });
-  uploader.upload({
-    fileInput,
-    complete: (response, error) => {
-      if (!error && response && response.url) {
-        success({
-          src: response.url
-        });
-      } else {
-        window.alert('There was a problem uploading the image: '+error);
-        failure();
-      }
-    }
-  });
-}
+const placeholderSrc = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAMFBMVEXp7vG6vsHm6+63u77Hy868wMPe4+bO09bh5unr8fTR1djAxMfM0NPX3N/c4eTBxcjXRf5TAAACh0lEQVR4nO3b6ZKqMBSFUSQMYZL3f9tbBq/NEEDiqUqOfusn1ZXKbjcQlGQZAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACC6RkbsGHuabChEtHmiGYfS3EQYM+Sxw/gMQvmcNnYaj6oTDHi73WPn2eqnj9B8zo3TJXcq5uNjXmVff86VwSR3JtryMa1BYqi7S1hJDCVpSigyLcGhJJEwzlCSNtPKrbVhVwsdCfOhH7uuaG3ARV9DwsaOzxt3N1yPqCHhvXytTUz92VDpmE/LLhZwl++R6Sds6sUa/PL6K/2E2fIhw1xdRKefsFolrPc+xNx/N0k/4fpBsdhL2HfeiN+TsDCms8dDpeRyS3P3QDl6Iqaf8L0rTf+80m6Lmn7Ct+4Wxf+/2RY1/YRv3PHz/u+fsCmqgoTnq7Z+8SGviqoh4dnKu1ieqauiakh4/PQ0r6ivqDoSHj0B97eNRVG1JNxV+L4bnxdVecJtRTdFVZ7QU9F1UXUn9FZ0VVRlCav5ob2KLouqKmFjy676u2HsVnRRVFUJq3J+8KCi86IqSthMvyl209Hjijqm3RsqAZ5pNfa5PJ2KelJRjQmr1/r7cfy0ouoSNvOfvbvhvKLaEr4qOin9kTQnrN7LpDZhE/Zmhp6Eq4p+YcKgiipKGFhRRQkDK6ooYfgLbiSMioQkJGF8P5XwHv4O+7AaKiXzaeXh1kMl5AffTUxiKEm/krD94BR8Gdxl1fceSlR58ZhXKbEpyD2amNiBtmrJLTMHL1LF8/rpXkSZXEmz8K8uvAFFNm6Iq0aBLUFOmeCuJ6exrcCmoLpN7kYx891bSAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADgh/wDdr8peyRHLogAAAAASUVORK5CYII=";
```

Comment:
what about having it in `utils/`? Having a file called `cards/placeholder-image.js` felt odd.